html-relative-links/projects/projects.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Projects</title>
+    <style>
+      img {
+        width: 300px;
+      }
+    </style>
+  </head>
+  <body>
+    <ul>
+      <li><a href="../index.html">Home Page</a></li>
+      <li><a href="../about-me/about-me.html">About Me</a></li>
+      <li><a href="#">Projects</a></li>
+      <li><a href="../contact-us/contact-us.html">Contact Us</a></li>
+    </ul>
+    <p>Hello Projects!</p>
+    <img
+      src="https://images.pexels.com/photos/6632422/pexels-photo-6632422.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2"
```

Comment:
This should be ponting to one of the images we provided for you in the `images` directory